xonsh/completer.py

```diff
@@ -467,10 +487,13 @@ def _load_cached_options(self):
         try:
             with open(OPTIONS_PATH, 'rb') as f:
                 self._options = pickle.load(f)
-        except:
+        except Exception:  # pylint: disable=broad-except
             self._options = {}
 
     def _save_cached_options(self):
         """Save completions to file."""
-        with open(OPTIONS_PATH, 'wb') as f:
-            pickle.dump(self._options, f)
+        try:
```

Comment:
Because I have moved the `try..except...` from the `ManCompleter.__del__` method. There did not appear to be any reason for the asymmetry between the `_load_cached_options` and `_save_cached_options` methods. If there is a reason for the asymmetry then that should be documented.